packages/datepicker/datepicker.scss

```diff
@@ -1,64 +1,163 @@
 @import "~@fremtind/jkl-core/variables/_all.scss";
 @import "~@fremtind/jkl-core/mixins/_all.scss";
 @import "~@fremtind/jkl-core/_functions.scss";
-@import "~@fremtind/jkl-select/select.scss";
 
-$transition-timing: 200ms cubic-bezier(0.7, 0, 0.3, 1);
+$datepicker-focus-color: $info;
+$datepicker-focus-color--inverted: $info--darkbg;
 
-$input-bg-color: inherit;
-$border-color: $svart;
-$border-color--focus: $focus-color;
+:root,
+[data-theme="light"] {
+    --datepicker-focus-color: #{$datepicker-focus-color};
+}
+[data-theme="dark"] {
+    --datepicker-focus-color: #{$datepicker-focus-color--inverted};
+}
+
+.jkl-datepicker {
+    @include reset-outline;
+    position: relative;
+
+    &__input-wrapper {
+        position: relative;
+    }
+
+    & &__input {
+        // adjust input padding for visual balance with numbers
+        padding-top: rem(10px);
+    }
+
+    &__calendar-wrapper {
+        position: absolute;
+        top: calc(100% + #{rem(14px)});
+        left: 0;
+        right: 0;
+    }
+
+    &--open {
+        .jkl-datepicker__input {
+            box-shadow: inset 0 0 0 rem(1px) var(--datepicker-focus-color, $datepicker-focus-color),
+                0 0 0 0 var(--datepicker-focus-color, $datepicker-focus-color);
```

Comment:
Jeg har ikke spesifisert noen fallback for IE11 her, men dersom #944 blir merget vil det bli lagt til automatisk av `postcss` under bygg.